src/models/supply-model.ts

```diff
@@ -7,17 +7,17 @@ export const Model =
         "type": "Timer"
       },
       {
-        "label": "n_fb_rur1",
+        "label": "nFbRur1",
```

Comment:
I am wondering if the researchers like @sam-pf look for labels like `n_fb_rur1` in their log analysis. If they do, we should let them know that the labels are changing.

Reply:
Hm, it might have been changed by accident when I did a global search and replace. It was mostly because TSLint was complaining about variable names. But we can definitely keep string labels with underscores.

Reply:
It is just a label, not a factor name, and I believe that the impact of the label change is very small, if any, on my part.  However, if it is just a linting issue, perhaps it can be made an exception and kept as is?  Thanks!

Reply:
I've just restored original labels.